fuse/moosefs/Portfile

```diff
@@ -0,0 +1,103 @@
+PortSystem		1.0
```

Comment:
Generally you want to start with a modeline comment; see the MacPorts guide for details.